responsibleai/tests/test_rai_insights.py

```diff
@@ -166,6 +166,39 @@ def test_modelanalysis_boston(self, manager_type):
             run_rai_insights(model, X_train, X_test, LABELS, ['CHAS'],
                              manager_type, manager_args)
 
+    def test_rai_insights_empty_save(self):
+        X_train, y_train, X_test, y_test, classes = \
+            create_binary_classification_dataset()
+
+        models = create_models_classification(X_train, y_train)
+        X_train[LABELS] = y_train
+        X_test[LABELS] = y_test
+
+        for model in models:
+            rai_insights = RAIInsights(
```

Comment:
nice test!

Reply:
It's largely a copy-paste of the others, tweaked to do the specific pattern of actions which revealed the bug....